src/styles/main.scss

```diff
@@ -1,3 +1,18 @@
+@import url("./stars.scss");
+@import url("./catalog.scss");
+@import url("./header.scss");
```

Comment:
the task says that you need to create a separete file for each BEM block(header, catalog, cars, stars)